sopsinstall.sh

```diff
@@ -9,15 +9,12 @@ usage() {
 
 $this: download binaries for mozilla/sops
```

Comment:
It looks like you might have lost the execute bit on this - was that intentional?

![screen shot 2018-03-12 at 14 54 29](https://user-images.githubusercontent.com/630449/37309032-479cd6e4-2605-11e8-9712-a69e0f44244f.png)

Reply:
doh nope. thanks.